server/cli/kv.go

```diff
@@ -122,14 +122,11 @@ func runPut(cmd *cobra.Command, args []string) {
 	if kvDB == nil {
 		return
 	}
-	err := kvDB.Tx(func(tx *client.Tx) error {
```

Comment:
Why did you make this change? I think that if you operate on multiple keys in one call to the CLI it should be done in one transaction.